Aztec/Classes/Libxml2/DOM/TextNode.swift

```diff
@@ -31,24 +31,41 @@ extension Libxml2 {
 
         // MARK: - EditableNode
         
-        func append(_ string: String) {
+        func append(_ string: String, undoManager: UndoManager? = nil) {
             contents.append(string)
         }
 
-        func deleteCharacters(inRange range: NSRange) {
+        func deleteCharacters(inRange range: NSRange, undoManager: UndoManager? = nil) {
 
             guard let textRange = contents.rangeFromNSRange(range) else {
                 fatalError("The specified range is out of bounds.")
             }
 
+            if let undoManager = undoManager, let range = contents.rangeFromNSRange(range) {
```

Comment:
Please, correct me if i'm wrong. The `let range = ` statement equals the one that's a couple lines above. Do we need this other `if let`  here?